src/screens/ChannelListScreen/elements/CustomPreviewMessage.js

```diff
@@ -28,7 +27,7 @@ const PreviewMessage = (props) => {
     props.latestMessagePreview?.messageObject?.isRemoveMember
   ) {
     return (
-      <Text numberOfLines={1} style={[styles.message, {color: COLORS.gray7}]}>
+      <Text numberOfLines={1} style={[styles.message, {color: '#7A7A7A'}]}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same issue as above with the hardcoded color value. Consider using a centralized color scheme.

```diff
- <Text numberOfLines={1} style={[styles.message, {color: '#7A7A7A'}]}>
+ <Text numberOfLines={1} style={[styles.message, {color: colors.gray7}]}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->